app/models/fe/date_field.rb

```diff
@@ -29,7 +29,7 @@ def response(answer_sheet = nil)
       r
     end
 
-    def display_response(answer_sheet = nil)
+    def display_response(answer_sheet = nil, humanize = false)
```

Comment:
I think rubocop would complain about this if it were running. I think unused parameters should start with `_`